go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/buger/goterm v1.0.4
 	github.com/buger/jsonparser v1.1.1
-	github.com/containerd/containerd v1.6.19
+	github.com/containerd/containerd v1.6.18
```

Comment:
Somehow, another dependency is being downgraded here. It's perhaps the case that your branch was taken before the dep was updated on main.

You can either undo the changes manually here in `go.mod` and the run a `go mod tidy`... or you could `git rebase upstream/main` to rebase the changes on top of the current state of main, and fix any conflicts that come up.